src/cookbooks/data_quality/collections.md

```diff
@@ -0,0 +1,34 @@
+## Collections in Bigeye
+Collections help you organize and focus on specific areas of interest, making it simpler to track and address data quality across different segments of your data landscape. This feature enhances efficiency by allowing users to monitor grouped entities in a cohesive manner.
+
+![](../../assets/Bigeye_screenshots/Bigeye-Collections.png)
+
+### Creating a new collection
+If you don’t find a collection that meets your product or requirements, admins can create a new collection.
+
+<video width="640" height="360" controls>
+        <source src="../../assets/Bigeye/creating_collection.mov" type="video/quicktime">
```

Comment:
mov files most likely will not be the best format to embed into a website. Plus those blow up the repo size considerably. 
It might be better to upload them to our youtube channel or even better to AirMo.